MonoGame.Framework.Content.Pipeline/Graphics/GraphicsUtil.cs

```diff
@@ -159,7 +167,9 @@ public static void CompressDxt(GraphicsProfile profile, TextureContent content,
 
             if (alphaRange == AlphaRange.Opaque)
                 Compress(typeof(Dxt1BitmapContent), content, generateMipMaps);
-            else if (alphaRange == AlphaRange.Cutout || sharpAlpha)
+            else if (sharpAlpha)
+                CompressFontDXT3(content, generateMipMaps);
```

Comment:
So maybe something very specific like `isSpriteFont` is better?  Or maybe `isMonochrome` or `isGreyscale`?